### DISCUSSION TITLE
Rosetta_cm protocol partial_thread.default.linuxgccrelease fault

### DISCUSSION THREAD

**Discussion by XHHnefu** [RosettaCommons/rosetta#210] [Rosetta]
Hello,

I've been trying to implement the Rosetta_cm protocol and I keep getting a fault error. Is there any idea what I might be doing incorrectly? The Rosetta Crash Log is below and is followed by the output logs (three raw text files are also included as the attachment).
[rosetta_cm fault.zip](https://github.com/user-attachments/files/17392755/rosetta_cm.fault.zip)

Thanks in advance for any help/assistance! 

##############################################################################################################
#
# Rosetta crash log. Please submit the contents of this file to http://crash.rosettacommons.org/
# or use the script Rosetta/main/source/scripts/python/public/report_crashes.py
#

[START_CRASH_REPORT]
[ROSETTA_VERSION]: 2024.09+release.06b3cf8
[COMMIT_DATE]: 2024-03-07T14:59:22.624665
[APPLICATION]: UNKNOWN APPLICATION
[MODE]: Release
[EXTRAS]: default
[OS]: GNU/Linux
[COMPILER]: GCC version "11.4.0"
[STDLIB]: libstdc++ version 20230528
[START_OPTIONS]
NO OPTIONS SPECIFIED

[END_OPTIONS]

[START_BACKTRACE]: RAW_LIBC
/home/xuhonghao/download/rosetta.source.release-371/main/source/build/src/release/linux/6.8/64/x86/gcc/11/default/libutility.so(backtrace_string[abi:cxx11](int)+0x5a) [0x78509bd7538a]
/home/xuhonghao/download/rosetta.source.release-371/main/source/build/src/release/linux/6.8/64/x86/gcc/11/default/libutility.so(utility::excn::Exception::Exception(char const*, int, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)+0xe0) [0x78509bcbcc00]
/home/xuhonghao/download/rosetta.source.release-371/main/source/build/src/release/linux/6.8/64/x86/gcc/11/default/libutility.so(+0x195ba6) [0x78509bb95ba6]
/home/xuhonghao/download/rosetta.source.release-371/main/source/build/src/release/linux/6.8/64/x86/gcc/11/default/libutility.so(utility::options::OptionCollection::load(int, char* const*, bool)+0x1db) [0x78509bbef60b]
/home/xuhonghao/download/rosetta.source.release-371/main/source/build/src/release/linux/6.8/64/x86/gcc/11/default/libcore.6.so(core::init::init_options(int, char**)+0x24) [0x78509fb4bc74]
/home/xuhonghao/download/rosetta.source.release-371/main/source/build/src/release/linux/6.8/64/x86/gcc/11/default/libcore.6.so(core::init::init(int, char**)+0x31) [0x78509fb4fed1]
/home/xuhonghao/download/rosetta.source.release-371/main/source/bin/partial_thread.default.linuxgccrelease(+0xb31d) [0x59771e8c131d]
/lib/x86_64-linux-gnu/libc.so.6(+0x29d90) [0x78509b229d90]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80) [0x78509b229e40]
/home/xuhonghao/download/rosetta.source.release-371/main/source/bin/partial_thread.default.linuxgccrelease(+0xdce5) [0x59771e8c3ce5]

[END_BACKTRACE]

##############################################################################################################
:~/Workspace/test/my_model$ /home/xuhonghao/download/rosetta.source.release-371/main/source/bin/partial_thread.default.linuxgccrelease /home/xuhonghao/download/rosetta.source.release-371/main/source/bin/partial_thread.default.linuxgccreleas -in:file:fasta 1u19.fasta -in:file:alignment 1u19_2RH1_A.grishin -in:file:template_pdb 2RH1_ISOLATED_A.pdb

[FILE]: src/utility/options/OptionCollection.cc
[LINE]: 347
[START_MESSAGE]
ERROR: Unused "free" argument specified: /home/xuhonghao/download/rosetta.source.release-371/main/source/bin/partial_thread.default.linuxgccreleas

[END_MESSAGE]
[END_CRASH_REPORT]

[ ERROR ]: Caught exception:

File: src/utility/options/OptionCollection.cc:347
ERROR: Unused "free" argument specified: /home/xuhonghao/download/rosetta.source.release-371/main/source/bin/partial_thread.default.linuxgccreleas

AN INTERNAL ERROR HAS OCCURED. PLEASE SEE THE CONTENTS OF ROSETTA_CRASH.log FOR DETAILS.

**Comment by roccomoretti** [comment]
`ERROR: Unused "free" argument specified: ` is what you get when you have an option on the command line that Rosetta doesn't understand. Specifically, it's an option which doesn't start with `-`.

The option in question is listed, and if you examine the command line you used, you'll see that the executable name is listed twice

> /home/xuhonghao/download/rosetta.source.release-371/main/source/bin/partial_thread.default.linuxgccrelease **_/home/xuhonghao/download/rosetta.source.release-371/main/source/bin/partial_thread.default.linuxgccreleas_** -in:file:fasta 1u19.fasta -in:file:alignment 1u19_2RH1_A.grishin -in:file:template_pdb 2RH1_ISOLATED_A.pdb

**Reply by XHHnefu** [reply]
Thank you！ That's very helpful.
